en/docs/reference/connectors/jira-connector/jira-connector-example.md

```diff
@@ -16,58 +16,58 @@ You will use two HTTP API resources, which are `createIssue` and `getIssue`.
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration project
 
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
+Follow the steps in the [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.
 
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+## Creating the Integration logic
 
-## Creating the Integration Logic
-
-1. Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API. 
-   <img src="{{base_path}}/assets/img/integrate/connectors/adding-an-api.jpg" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+1. Click `+` on the Extension panel APIs to create the REST API.
 
 2. Provide the API name as `jiraAPI` and the API context as `/jira`. You can go to the source view of the XML configuration file of the API and copy the following configuration.
+   ![Adding a Rest API]({{base_path}}/assets/img/integrate/connectors/jira-conn-1.png "Adding a Rest API")
+
+3. Click the `/resource` default endpoint to open the **Resource View**. Then click the `+` arrow below the Start node to open the side panel. Select **Connectors** and search for the **jira** connector. Click the Jira connector to open the operation panel, then click **init**. This will download the connector.
```

Comment:
```suggestion
3. Click the `/resource` default endpoint to open the **Resource View**. Then click the `+` arrow below the Start node to open the side panel. Select **Connectors** and search for the **Jira** connector. Click the Jira connector to open the operation panel, then click **init**. This will download the connector.
```